.travis.yml

```diff
@@ -3,6 +3,9 @@ rvm:
   - 2.2.5
   - 2.3.1
 
+env:
+  - I3S_AUTH_FILE=examples/login_for_integration_i3s_tests.json
```

Comment:
@fgbulsoni I believe that env var should be set via Travis repository settings, but I'm not able to change it.